apps/vc-api/src/vc-api/credentials/credentials.service.ts

```diff
@@ -74,21 +74,39 @@ export class CredentialsService implements CredentialVerifier {
       issueDto.options,
       verificationMethod.id
     );
-    return JSON.parse(
-      await issueCredential(
-        JSON.stringify(issueDto.credential),
-        JSON.stringify(proofOptions),
-        JSON.stringify(key)
-      )
-    );
+
+    const result = await issueCredential(
+      JSON.stringify(issueDto.credential),
+      JSON.stringify(proofOptions),
+      JSON.stringify(key)
+    ).catch((err) => {
+      if (typeof err === 'string') {
+        //TODO: discuss if BadRequestException should be thrown here instead?
+        throw new InternalServerErrorException(`@spruceid/didkit-wasm-node.issueCredential error: ${err}`);
```

Comment:
I agree. My only concern is that potentially error can be thrown because something malfunctioned due to the library error, not an incorrect request. I have no idea how to detect this here.